### PR TITLE
[RPS-430] Merge sitemaps and move /publications

### PR DESCRIPTION
**Pull request by jacsamell**
Site maps have been merged to allow for /publications to go under
/data-and-information.
Removed publications overview as it is not longer needed.
Also added a redirect rule so /publications links will still work.

### CODE REVIEW

File: acceptance-tests/src/test/resources/features/site/urlrewrite.feature [hunk 1]
```diff
@@ -4,3 +4,8 @@ Feature: As a content editor I need to set up URL redirects so legacy documents
     Scenario: Redirect to shmi hub page
         When I navigate to the "shmi" page
         Then I should see page titled "Summary Hospital-level Mortality Indicator (SHMI)"
+
+    Scenario: publications redirect
```
Comment (mchalejeiwc) [inline acceptance-tests/src/test/resources/features/site/urlrewrite.feature]: Nice

File: docs/what-if/use-custom-host.md [hunk 2]
```diff
@@ -14,10 +14,6 @@ definitions:
     /hst:hst/hst:hosts/localhost-two:
       /127.0.0.2:
         /hst:root:
-          /publications:
```
Comment (wojtek-oledzki) [inline docs/what-if/use-custom-host.md]: 👍

File: repository-data/application/src/main/resources/hcm-config/hst/configurations/common/sitemap.yaml [hunk 3]
```diff
@@ -72,107 +51,82 @@ definitions:
...
-      /hubs:
-        /_any_:
-          /_index_:
+        /publications:
+          /_any_:
+            /_index_:
+              hst:componentconfigurationid: hst:pages/404
+              hst:componentconfigurationmappingnames:
+              - publicationsystem:archive
+              - publicationsystem:dataset
```
Comment (wojtek-oledzki) [inline repository-data/application/src/main/resources/hcm-config/hst/configurations/common/sitemap.yaml]: we probably only need:
```
- publicationsystem:archive
- publicationsystem:legacypublication
- publicationsystem:publication
- publicationsystem:series
```

File: repository-data/application/src/main/resources/hcm-config/hst/hosts.yaml [hunk 4]
```diff
@@ -11,14 +11,6 @@ definitions:
...
                     /hst:root:
-                      /indicators:
-                        hst:homepage: root
-                        hst:mountpoint: /hst:hst/hst:sites/nationalindicatorlibrary
-                        jcr:primaryType: hst:mount
-                      /publications:
-                        hst:homepage: root
-                        hst:mountpoint: /hst:hst/hst:sites/publicationsystem
-                        jcr:primaryType: hst:mount
                       hst:alias: common-context
```
Comment (wojtek-oledzki) [inline repository-data/application/src/main/resources/hcm-config/hst/hosts.yaml]: We can get rid of the alias from here and ftl's

**Pull request merged**